sumpy/expansion/__init__.py

```diff
@@ -282,12 +282,11 @@ def _split_coeffs_into_hyperplanes(self) -> List[Tuple[int, List[Tuple[int]]]]:
                 pass
             else:
                 # Calculate the multi-index that appears last in in the PDE in
-                # reverse degree lexicographic order (degrevlex).
-                # The monomial ordering used here which is degrevlex, must match
-                # the monomial ordering used in LinearPDEBasedExpansionTermsWrangler
-                max_mi_idx = max(mi_to_index[ident.mi] for
-                                 ident in pde_dict.keys())
-                max_mi = mis[max_mi_idx]
+                # degree lexicographic order.
+                ordering_key = self._get_mi_ordering_key()
+                pde_dict, = self.knl.get_pde_as_diff_op().eqs
```

Comment:
- The same line is just above. Does it need to be repeated?
- Renaming `pde_dict` to `deriv_id_to_coeff` would be easier to read IMO.

Reply:
Removed the line as it was a duplicate.
Rename done in 4b26ae0